VotingApplication/VotingApplication.Web/Views/Routes/Vote.cshtml

```diff
@@ -6,6 +6,9 @@
     </div>
     <div voting-strategy></div>
     <div class="flexbox">
-        <a class="btn inactive-btn centered" ng-href="{{resultsLink}}">Go to results</a>
+        <div class="centered">
+            <a class="btn inactive-btn" ng-href="{{resultsLink}}">Go to results</a>
```

Comment:
The buttons need to be on different lines, and preferably the same width.

Ideally on multi-option votes (all but Basic) we would have;

Vote
Clear votes
Go to results